vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "AccessibilityInfoModule.h"
+#include <UI.Xaml.Automation.Peers.h>
+#include <UI.Xaml.Controls.h>
+#include <XamlUtils.h>
+#include <winrt/Windows.ApplicationModel.DataTransfer.h>
+#include <winrt/Windows.UI.ViewManagement.h>
+#include "Unicode.h"
+#include "Utils/Helpers.h"
+
+namespace Microsoft::ReactNative {
+
+void AccessibilityInfo::Initialize(winrt::Microsoft::ReactNative::ReactContext const &reactContext) noexcept {
+  m_context = reactContext;
+}
+
+void AccessibilityInfo::isReduceMotionEnabled(std::function<void(React::JSValue const &)> const &onSuccess) noexcept {
+  auto jsDispatcher = m_context.JSDispatcher();
+  m_context.UIDispatcher().Post([weakThis = weak_from_this(), jsDispatcher, onSuccess] {
+    if (auto strongThis = weakThis.lock()) {
+      winrt::Windows::UI::ViewManagement::UISettings uiSettings;
+      auto animationsEnabled = uiSettings.AnimationsEnabled();
+      jsDispatcher.Post([animationsEnabled, onSuccess] { onSuccess(!animationsEnabled); });
+    }
+  });
+}
+
+void AccessibilityInfo::isTouchExplorationEnabled(
+    std::function<void(React::JSValue const &)> const &onSuccess) noexcept {
+  onSuccess(false);
+}
+
+void AccessibilityInfo::setAccessibilityFocus(double reactTag) noexcept {
```

Comment:
Can't remember if we generate warnings for unused params or not, but consider commenting out the name.